### PR TITLE
Arch-Install-Readme

### PR DESCRIPTION
**Pull request by Data-007**
Updated the Istallation instructions. Added in pipx.

### CODE REVIEW

File: docs/installation/arch.md [hunk 1]
```diff
@@ -29,9 +23,9 @@ sudo udevadm trigger
 ```
 If the software is having problems later to detect the Stream Deck, you can try unplugging/plugging it back in.
 
-## Install Stream Deck UI
-```
-python -m pip install streamdeck-ui --user
+## Install Streamdeck
+```console
+sudo pipx install  streamdeck-linux-gui
```
Comment (4s3ti) [inline docs/installation/arch.md:28]: don't use sudo. not good practice.

**Review by 4s3ti — CHANGES_REQUESTED** [review]
change the target branch to develop as per https://github.com/streamdeck-linux-gui/streamdeck-linux-gui#general-guidelines

**Comment by Data-007** [comment]
Ive removed sudo my end cant figure how to update here in this pull request

**Comment by 4s3ti** [comment]
1 - `git reset main` 
2 - make the necessary changes
3 - `git commit -a -m docs: update arch install docs`
4 - git push origin main --force

and then this pull request should be updated automatically and show the changes.

**Pull request closed without merging**